package.json

```
@@ -10,11 +10,19 @@
   },
   "dependencies": {
     "axios": "^0.26.1",
+    "bootstrap": "^5.1.3",
     "dotenv": "^16.0.0",
     "next": "^11.1.3",
+<<<<<<< HEAD
+
+    "next-images": "^1.8.4",
+
+=======
+>>>>>>> bb4a1ee5dbc7ede1734099681795425248e0844b
```

Please resolve properly.
```suggestion
    "next-images": "^1.8.4",
```